streamlit_dashboard/dashboard.py

```diff
@@ -592,6 +209,52 @@ def get_user_name(user_id: int):
         return None
 
 
+def show_track_page():
```

Comment:
Gorgeous, thank you for refactoring this into different functions, makes the code way more readable